qutebrowser/completion/completer.py

```diff
@@ -90,7 +90,12 @@ def _get_new_completion(self, before_cursor, under_cursor):
             log.completion.debug('Starting command completion')
             return miscmodels.command
         try:
-            cmd = objects.commands[before_cursor[0]]
+            cmdname = before_cursor[0]
+            if before_cursor[0] in config.cache['aliases']:
+                cmdname = config.cache['aliases'][before_cursor[0]].split(
+                    maxsplit=1)[0]
```

Comment:
You should only need one dict lookup here, something like:

```suggestion
            cmdname = before_cursor[0]
            alias = config.cache['aliases'].get(cmdname)
            if alias: 
                cmdname = alias.split(maxsplit=1)[0]
```